Jenkinsfile-test

```diff
@@ -2,7 +2,7 @@ library("tdr-jenkinslib")
 
 def repo = "tdr-antivirus"
 def versionTag = "v${env.BUILD_NUMBER}"
-def yaraVersion = "4.1.0"
+def yaraVersion = "4.1.3"
```

Comment:
Is there anyway to define the yara version in a single place? Its being defined in multiple files.

Reply:
I've added in a yara-version.txt file which both of the Jenkinsfiles read from and this looks like it's working so we'd only have to change one file. 

The other place it's in is in the requirements txt file for yara-python. I can't see a way of reading this globally. It's the version of the python library though and not the version of yara itself so I think it's ok to keep it separate, although they should be kept in sync really
The other thing to do with the python version is just to remove the version and always use the latest but that might cause  other problems down the line.